base/random.jl

```diff
@@ -1166,6 +1166,29 @@ immutable UUID
 end
 UUID(u::AbstractString) = convert(UUID, u)
```

Comment:
Actually I couldn't convert from a string to UUID after removing that -- pardon my ignorance, but is that a super recent change?